components/cli/pkg/constants/const.go

```diff
@@ -100,6 +102,10 @@ const CELLERY_EXECUTABLE_PATH = "/runtime/executable/"
 const BALLERINA_INSTALLATION_PATH_MAC = "/Library/Ballerina/ballerina-0.991.0"
 const BALLERINA_INSTALLATION_PATH_UBUNTU = "/usr/lib/ballerina/ballerina-0.991.0"
 const BALLERINA_EXECUTABLE_PATH = "/bin/"
+const BALLERINA_CONF = "ballerina.conf"
+const BALLERINA_TOML = "Ballerina.toml"
+const BALLERINA_LOCAL_REPO = ".ballerina/"
```

Comment:
shouldn't this be ~/.ballerina/ ?

Reply:
No, it resides in the ballerina project directory.